ts/components/OnionStatusDialog.tsx

```diff
@@ -25,6 +24,15 @@ import {
   getOnionPathsCount,
 } from '../state/selectors/onions';
 
+// tslint:disable-next-line: no-submodule-imports
+import useNetworkState from 'react-use/lib/useNetworkState';
+
+export type OnionPathModalType = {
+  confirmText?: string;
+  cancelText?: string;
+  title?: string;
+};
+
```

Comment:
is this used anywhere?

Reply:
this shouldn't, you should just call directly the window.i18n needed in the dialog itself